sources/packages/web/src/views/institution/Login.vue

```diff
@@ -33,6 +33,10 @@
       Disabled User - you dont have access to the system. Please contact
       Administrator for more informations.
     </Message>
+    <Message severity="error" class="mx-2" v-if="unknownUser">
+      Institution already exists. Please contact Administrator for more

```

Comment:
I believe that we could be a bit more clear about the fact that the user exists and it is valid on BCeID but he is not allowed to access the application.
I would recommend something lie below:

_"The user was validated with success but it is not currently allowed to have access to this application. Please contact the Administrator for more information."_

Reply:
Thanks. This make more sense. I will change to this.(with slight modification)
"The user was validated successfully but not currently allowed to have access to this application. Please contact the Administrator for more information."

Reply:
If that is ok

Reply:
I liked it 👍 
I believe that there is just a "is" missing in the middle of it. Either way, it sounds good to me.
_"The user was validated successfully but is not currently allowed to have access to this application. Please contact the Administrator for more information."_

Reply:
Verbiage modified.